test/fixtures/templates/bad/custom_rule_invalid_op.txt

```diff
@@ -0,0 +1 @@
+AWS::IAM::Role AssumeRolePolicyDocument.Version EQUALING "2012-10-15"
```

Comment:
think the rest of the files in [`test/fixtures/templates/`](https://github.com/aws-cloudformation/cfn-python-lint/tree/master/test/fixtures/templates) are CloudFormation templates, so probably a bit more organized to separate these out into a separate directory in [`test/fixtures/`](https://github.com/aws-cloudformation/cfn-python-lint/tree/master/test/fixtures) or something

Reply:
Will do!